docker-compose.yml

```diff
@@ -19,6 +19,10 @@ services:
       - .env
     ports:
       - "5432:5432"
+    environment:
```

Comment:
polish: explicitly declare these variables is not needed because the environment variables are already defined in the `.env` file.